src/components/learner-credit-management/SubBudgetCard.jsx

```diff
@@ -116,88 +136,94 @@ const BaseSubBudgetCard = ({
+    }
+
+    return (
+      <Card.Section
+        title={!isRetired && (
+          <h4>
             <FormattedMessage
-              id="lcm.budgets.budget.card.available"
-              defaultMessage="Available"
-              description="Label for the available balance on the budget card"
+              id="lcm.budgets.budget.card.balance"
+              defaultMessage="Balance"
+              description="Header for the balance section of the budget card"
             />
-          </div>
-          <span className="small">
-            {isFetchingBudgets ? <Skeleton /> : formatPrice(available)}
-          </span>
-        </Col>
-        {isAssignable && (
-          <Col xs="6" md="auto" className="mb-3 mb-md-0">
-            <div className="small font-weight-bold">
+          </h4>
+        )}
+        muted
+      >
+        <Col className="d-flex justify-content-start w-md-75">
+          {!isRetired && (
+            <>
+              <Col xs="6" md="auto" className="mb-3 mb-md-0 ml-n4.5">
+                <div className="small font-weight-bold">
+                  <FormattedMessage
+                    id="lcm.budgets.budget.card.available"
+                    defaultMessage="Available"
+                    description="Label for the available balance on the budget card"
+                  />
+                </div>
+                <span className="small">
+                  {isFetchingBudgets ? <Skeleton /> : formatPrice(available)}
+                </span>
+              </Col>
+              {isAssignable && (
+                <Col xs="6" md="auto" className="mb-3 mb-md-0">
+                  <div className="small font-weight-bold">
+                    <FormattedMessage
+                      id="lcm.budgets.budget.card.assigned"
+                      defaultMessage="Assigned"
+                      description="Label for the assigned balance on the budget card"
+                    />
+                  </div>
+                  <span className="small">
+                    {isFetchingBudgets ? <Skeleton /> : formatPrice(pending)}
+                  </span>
+                </Col>
+              )}
+            </>
+          )}
+          <Col xs="6" md="auto" className={classNames('mb-3 mb-md-0', { 'ml-n4.5': isRetired })}>
+            <div className={classNames('font-weight-bold', { h4: isRetired, small: !isRetired })}>
               <FormattedMessage
-                id="lcm.budgets.budget.card.assigned"
-                defaultMessage="Assigned"
-                description="Label for the assigned balance on the budget card"
+                id="lcm.budgets.budget.card.spent"
+                defaultMessage="Spent"
+                description="Label for the spent balance on the budget card"
               />
             </div>
-            <span className="small">
-              {isFetchingBudgets ? <Skeleton /> : formatPrice(pending)}
+            <span className={classNames({ small: !isRetired })}>
```

Comment:
[inform] As it turns out, because the budget cards are rendered within a `DataTable`, the default font size within the cards is always `small` (14px). The Figma spec calls for 18px (default body font size). However, there is no existing CSS utility class that will override the default `DataTable` font size styles back to the default font size of 18px.

@zwidekalanga As the final thing for this PR, do you mind adding the following to the bottom of `src/index.scss`? Ideally this sort of CSS utility class would be in Paragon, but for now we can define our own. I will propose to the Paragon Working Group tomorrow (5/22) about contributing the `font-size-base` (and/or similar other font-sizes) CSS utility classes upstream as it relates to some of the discussion [here](https://github.com/openedx/open-edx-proposals/pull/580) about CSS best practices.

```scss
.font-size-base {
  font-size: $font-size-base !important;
}
```

```jsx
<span className={classNames({ small: !isRetired, 'font-size-base': isRetired })}>
```

Reply:
I'll hold off on merging now as I will be heading out for the day soon. I'll check back in the morning to merge and will inform you when it lands on the staging environment.